Api/Client.php

```diff
@@ -89,6 +101,7 @@ public function setApiDetails($refreshToken, $clientId, $secret, $redirectUri, $
             'refresh_token' => $refreshToken,
             'dev_token' => $devToken,
         ];
+        return $this;
```

Comment:
It should at least be consistent, either everything should be fluent, or not, but now it's half fluent / half not-fluent. So let's make it consistent (probably not fluent, unless we have a good reason to make it fluent)